lib/data/external/AwsClient.js

```diff
@@ -208,6 +208,23 @@ class AwsClient {
             });
         });
     }
+
+    abortMPU(key, uploadId, bucket, log, callback) {
+        const awsBucket = this._awsBucketName;
+        const awsKey = this._createAwsKey(bucket, key, this._bucketMatch);
+        const abortParams = {
+            Bucket: awsBucket, Key: awsKey, UploadId: uploadId,
+        };
+        return this._client.abortMultipartUpload(abortParams, err => {
+            if (err) {
+                log.error('There was an error aborting the MPU on AWS S3. You' +
+                    ' should abort directly on AWS S3 using the same uploadId.',
+                { error: err, dataStoreName: this._dataStoreName });
```

Comment:
does this error actually show up in the logs or do you have to do err.message and err. stack?